app/scripts/modules/ecs/src/ecs.help.ts

```diff
@@ -60,6 +60,8 @@ const helpContents: { [key: string]: string } = {
   'ecs.containerMappingName':
     '<p>The name of the container. Name should match the <a href="https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDefinition.html#ECS-Type-ContainerDefinition-name"><b>containerDefinition.name</b></a> field as it appears in the Task Definition.</p>',
   'ecs.containerMappingImage': '<p>The container image the named container should run.</p>',
+  'ecs.targetGroupMappings':
+    '<p>The list of expected target groups within the Task Definition and which port they should map to within the specified container.</p>',
```

Comment:
I'm confused by this explanation. What do you mean by target groups within the task definition? Target groups are specified at the service level...

Reply:
Updated

Reply:
I still find this confusing.  Is this better?  

'The list of target groups through which the ECS service will receive load balancer traffic.  Each target group is mapped to a container name and port within the Task Definition to specify which container should be registered to the target group.'

Reply:
yeah, the phrase "...target groups within the Task Definition" is the crux of the confusion. +1 to above reword

Reply:
Addressed.